airflow/operators/singularity_operator.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
```

Comment:
All new operators should be added to `airflow/contrib/operators/`.

Reply:
Fixed. Example also updated -> https://github.com/researchapps/airflow-example/blob/master/singularity/pipeline.py#L4